claim-check/code-samples/sample-2/client-consumer/ClientConsumer.csproj

```diff
@@ -8,8 +8,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.EventHubs" Version="2.2.1" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="9.4.1" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.6" />
```

Comment:
The GA packages have been released,  you'll likely want to bump these to `5.0.1`